src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+  let total = 0;
+
+  if (days < 3) {
+    total = days * perDay;
```

Comment:
To simplify the code and calculations, you can firstly calculate the total cost and in the `if` statements return the value with discount

` days * perDay` is repeating 3 times